features/clone.feature

```diff
@@ -286,3 +286,17 @@ Feature: hub clone
     When I successfully run `hub clone rtomayko/ronn`
     Then it should clone "git://github.com/RTomayko/ronin.git"
     And there should be no output
+
+  Scenario: If alias named clone exists, it should not be expanded.
+      Given the GitHub API server:
+        """
+        get('/repos/rtomayko/ronn') {
+          json :private => false,
+               :name => 'ronn', :owner => { :login => 'rtomayko' },
+               :permissions => { :push => false }
+        }
+        """
+      When I successfully run `git config --global alias.clone "clone -v"`
```

Comment:
No.
If alias is expanded, command run will be `git clone -v git://github.com/rtomayko/ronn.git\n`. However, step `Then it should clone "git://github.com/rtomayko/ronn.git"` expects that command run is actually `git clone git://github.com/rtomayko/ronn.git\n`. If alias is not expanded (which is success behavior for this test), the latter command is run.